lib/mix/tasks/surface/surface.format.ex

```diff
@@ -0,0 +1,499 @@
+defmodule Mix.Tasks.Surface.Format do
```

Comment:
I'm not sure if we want to pull in this mix task. It's a _lot_ of code copied over from `Mix.Tasks.Format`, so it feels a bit hacky to keep it around now that Formatter Plugins are available. However, not everyone is going to immediately switch over to Elixir 1.13 as soon as it's available, so it may be best to keep this.

Reply:
Agreed. Let's keep it for for a while.